subscriber/src/subscriber/tests/main_test.bal

```diff
@@ -0,0 +1,108 @@
+import ballerina/file;
+import ballerina/io;
+import ballerina/log;
+import ballerina/test;
+
+// This test asserts the calculated values in the distribution flow.
+// Test source : Test driver/FakeElection
+// Before enabling the test, make sure the aforementioned test is executed and json files exist in the subscriber root
+@test:Config { enable : false }
+function testReceivedDataViaTesDriverFakeElection() {
+    testR_VI("0002-R_VI-ED-01--Colombo.json", 15, 65, 10, 90, 10, 100, 150);
+    testR_VI("0004-R_VI-ED-01--Colombo.json", 105, 100, 35, 240, 30, 270, 350);
+    testR_VI("0006-R_VI-ED-01--Colombo.json", 115, 100, 47, 262, 33, 295, 380);
+    testR_VI("0008-R_VI-ED-02--Gampaha.json", 10, 50, 20, 80, 10, 90, 100);
+    testR_VI("0010-R_VI-ED-02--Gampaha.json", 60, 70, 40, 170, 20, 190, 225);
+
+    //testNational("0012-R_SI-N.json", "Foo", 10, "Bar", 5, "Baz", 3);
+    //testNational("0014-R_SI-N.json", "Bar", 25, "Foo", 18, "Baz", 4);
```

Comment:
```suggestion
    testNational("0012-R_SI-N.json", "Foo", 10, "Bar", 5, "Baz", 3);
    testNational("0014-R_SI-N.json", "Bar", 25, "Foo", 18, "Baz", 4);
```

Since we merged https://github.com/ECLK/Results-Dist/pull/147